src/lib/CollectStore.js

```diff
@@ -7,10 +7,18 @@ import * as konnectors from './konnectors'
 
 const INSTALL_TIMEOUT = 120 * 1000
 
+const sortByName = (a, b) => {
```

Comment:
I agree but I would suggest to mutualise it when we'll need to use it a second time.